app/controllers/policy_assessments_controller.rb

```diff
@@ -0,0 +1,65 @@
+class PolicyAssessmentsController < PlanningApplicationsController
+  before_action :set_planning_application
+  before_action :set_policy_class, only: %i[show_class update_class]
+
+  def part; end
+
+  def new
+    @part = params[:part]
+
+    redirect_to part_new_policy_assessment_path(@planning_application), alert: "Please choose one of the policy parts" if @part.blank?
+  end
+
+  def create
```

Comment:
Do we need to restrict `create` to planning applications that are `in_assessment` only?

Reply:
Yes that's a good shout